.evergreen/config/functions.yml

```diff
@@ -0,0 +1,355 @@
+        remote_file: ${project}/${FETCH_BUILD_VARIANT}/${revision}/${FETCH_BUILD_TASK}/${version}.tar.gz
+        local_file: build.tar.gz
+    - command: archive.targz_extract
+      params:
+        destination: src
+        path: build.tar.gz
+    # Fix path names in Makefile
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: "src"
+        script: |
+          ${PREPARE_SHELL}
+          perl -p -i -e "s|/data/mci/[0-9a-z]+/src|${PROJECT_DIRECTORY}|g" Makefile
+
+  "compile driver" :
+    - command: shell.exec
+      params:
+        working_dir: "src/src/libmongoc"
+        add_expansions_to_env: true
+        script: |
+          if [ -n "${LIBMONGOC_VERSION}" ]; then
+             echo "Checking out libmongoc version: ${LIBMONGOC_VERSION}"
+             git fetch
+             git checkout ${LIBMONGOC_VERSION}
+             # Note: compile-unix.sh will run `make libmongoc-version-current`
+          fi
+    - command: subprocess.exec
+      type: test
+      params:
+        working_dir: "src"
+        add_expansions_to_env: true
+        binary: bash
+        args:
+          - ${PROJECT_DIRECTORY}/.evergreen/compile-unix.sh
+
+  "upload mo artifacts":
+    - command: shell.exec
+      params:
+        script: |
+          ${PREPARE_SHELL}
+          find $MONGO_ORCHESTRATION_HOME -name \*.log | xargs tar czf mongodb-logs.tar.gz
+    - command: s3.put
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_file: mongodb-logs.tar.gz
+        remote_file: ${UPLOAD_BUCKET}/${build_variant}/${revision}/${version_id}/${build_id}/logs/${task_id}-${execution}-mongodb-logs.tar.gz
+        bucket: mciuploads
+        permissions: public-read
+        content_type: ${content_type|application/x-gzip}
+        display_name: "mongodb-logs.tar.gz"
+    - command: s3.put
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_file: ${DRIVERS_TOOLS}/.evergreen/orchestration/server.log
+        remote_file: ${UPLOAD_BUCKET}/${build_variant}/${revision}/${version_id}/${build_id}/logs/${task_id}-${execution}-orchestration.log
+        bucket: mciuploads
+        permissions: public-read
+        content_type: ${content_type|text/plain}
+        display_name: "orchestration.log"
+    - command: shell.exec
+      params:
+        script: |
+          ${PREPARE_SHELL}
+          find /tmp/MO -name \*.log | xargs tar czf mongodb-start-logs.tar.gz
+    - command: s3.put
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_file: mongodb-start-logs.tar.gz
+        remote_file: ${UPLOAD_BUCKET}/${build_variant}/${revision}/${version_id}/${build_id}/logs/${task_id}-${execution}-mongodb-start-logs.tar.gz
+        bucket: mciuploads
+        permissions: public-read
+        content_type: ${content_type|application/x-gzip}
+        display_name: "mongodb-start-logs.tar.gz"
+
+  "upload working dir":
```

Comment:
Noted that you proposed deleting this in https://github.com/mongodb/mongo-php-driver/pull/1473/files#r1338184023.

Are there other irrelevant functions? Perhaps `"upload mo artifacts"`? I assume that has to do with server logs, but I can't remember ever needing that.

Reply:
Removed a number of other unnecessary functions, including "upload mo artifacts".